src/components/Tutorial/TitleDescription.jsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+const TitleDescription = ({
+  color,
+  num,
+  title,
+  description,
+  component: Component,
+}) => {
+  return (
+    <>
+      <SContentboxContainer>
+        <STitleBoxContainer>
+          <SNumWrapper color={color}>{num}</SNumWrapper>
+          <STitleWrapper>{title}</STitleWrapper>
+        </STitleBoxContainer>
+        <SExplainBoxWrapper>
+          <SExplainWrapper>{description}</SExplainWrapper>
+        </SExplainBoxWrapper>
+        {Component && <Component />}
+      </SContentboxContainer>
+    </>

```

Comment:
`SContentboxContainer`로 가장 바깥을 감싸고 있으니 빈 태그로 한번더 감싸지 않아도 될 것 같아요~

```suggestion
    <SContentboxContainer>
      <STitleBoxContainer>
        <SNumWrapper color={color}>{num}</SNumWrapper>
        <STitleWrapper>{title}</STitleWrapper>
      </STitleBoxContainer>
      <SExplainBoxWrapper>
        <SExplainWrapper>{description}</SExplainWrapper>
      </SExplainBoxWrapper>
      {Component && <Component />}
    </SContentboxContainer>
```